### PR TITLE
Lm aw 144 145 146 147 148

### PR DESCRIPTION
**Pull request by LarryMatte**
supportarcbees-en
supportarcbees-fr

### CODE REVIEW

File: src/main/java/com/arcbees/website/client/application/support/ScrollToAnimation.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package com.arcbees.website.client.application.support;
+
+import com.google.gwt.animation.client.Animation;
+import com.google.gwt.dom.client.Element;
+import com.google.gwt.query.client.plugins.effects.PropertiesAnimation;
+
+public class ScrollToAnimation extends Animation {
+   private final Element element;
+
+   private int target;
```
Comment (meriouma) [inline src/main/java/com/arcbees/website/client/application/support/ScrollToAnimation.java]: Can be final

File: src/main/java/com/arcbees/website/client/application/support/SupportView.java [hunk 2]
```diff
@@ -16,18 +16,45 @@
...
+import static com.google.gwt.query.client.GQuery.$;
+
 public class SupportView extends ViewImpl implements SupportPresenter.MyView {
     interface Binder extends UiBinder<Widget, SupportView> {
     }
 
+    @UiField
+    AnchorElement supportPackage;
+    @UiField
+    Element supportPackage2;
```
Comment (meriouma) [inline src/main/java/com/arcbees/website/client/application/support/SupportView.java]: Rename to something more meaningful

File: src/main/java/com/arcbees/website/client/application/ui/supportcarousel/SupportCarousel.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+import static com.google.gwt.query.client.GQuery.$;
+
+public class SupportCarousel implements IsWidget {
+    interface Binder extends UiBinder<HTMLPanel, SupportCarousel> {
+    }
+
+    private static Binder BINDER = GWT.create(Binder.class);
+
+    private final HTMLPanel root;
```
Comment (meriouma) [inline src/main/java/com/arcbees/website/client/application/ui/supportcarousel/SupportCarousel.java]: `private final` field should after the UiFields

File: src/main/resources/com/google/gwt/i18n/client/LocalizableResource_fr.properties [hunk 4]
```diff
@@ -219,6 +207,24 @@ F13F0ED4CC01CE176DC047EBB5764122=développeurs
...
 A687023D891D8E16F16913977EAD338D=<strong>499,95\$</strong> par mois
 
+# Description: Video Conference
+2987ACF88562163A04157BBF8F69B544=Vidéo <span>Conférence</span>
+
+# Description: Knowledge Base - Paragraph -
+6BD06A7EADEB450FFD4F6885C5655BDB=Ayez toujours à portée de main les réponses à vos questions-clés grâce à notre base de connaissances.
+
+# Description: Purchasing support packages...
+75AC160BD20C2821FA23C10581A1CFEE=<p>En adhérant à notre Programme de soutien, non seulement vous donnez les outils à votre équipe afin qu''elle soit productive de façon optimale, mais vous pouvez dormir sur vos deux oreilles, sachant qu''en cas de publication d''une nouvelle version de notre logiciel, nous allons supporter la version précédente pendant 2 années supplémentaires.</p> <p>Puisque la majorité de nos outils <em lang="en">open source</em> sont inclus dans notre Programme de soutien, votre développement stack demeure stable pendant que vous déployez votre code en production et que vous profitez des résultats réels de votre entreprise.</p>
```
Comment (meriouma) [inline src/main/resources/com/google/gwt/i18n/client/LocalizableResource_fr.properties]: Someone has a good translation for `development stack`?

File: src/main/java/com/arcbees/website/client/application/support/ScrollToAnimation.java [hunk 5]
```diff
@@ -0,0 +1,42 @@
+package com.arcbees.website.client.application.support;
```
Comment (christiangoudreau) [inline src/main/java/com/arcbees/website/client/application/support/ScrollToAnimation.java]: missing license header

File: src/main/resources/com/google/gwt/i18n/client/LocalizableResource_fr.properties (hunk 4 above)
Comment (christiangoudreau) [inline src/main/resources/com/google/gwt/i18n/client/LocalizableResource_fr.properties]: Stack de développement? lol

File: src/main/java/com/arcbees/website/client/application/support/ScrollToAnimation.java [hunk 6]
```diff
@@ -0,0 +1,58 @@
+/*
```
Comment (jasonlemay) [inline src/main/java/com/arcbees/website/client/application/support/ScrollToAnimation.java]: @meriouma @christiangoudreau Shouldn't this thing be a plugin? I see the ScrollToAnimation as something that we could need in every projects, and I find its current location wrong ... Should be more like a widget at least, no?

Comment (jasonlemay) [inline src/main/java/com/arcbees/website/client/application/support/ScrollToAnimation.java]: "widget" might not be the appropriate word here

File: src/main/java/com/arcbees/website/client/application/support/SupportView.ui.xml [hunk 7]
```diff
@@ -215,20 +221,22 @@
                     <ui:msg description="Features">Features</ui:msg>
                 </h2>
                 <div class="{res.grid.row} {res.style.clearfix} {res.style.wrapper}">
-                    <div class="{res.grid.col} {res.grid.col_1_4} {res.grid.col_t_1_3} {res.grid.col_t_push_0} {res.grid.col_m_full} {res.grid.col_m_push_0}">
+                    <div class="{res.grid.col} {res.grid.col_1_4} {res.grid.col_t_1_3} {res.grid.col_t_push_0}
+                    {res.grid.col_m_full} {res.grid.col_m_push_0}">
```
Comment (jasonlemay) [inline src/main/java/com/arcbees/website/client/application/support/SupportView.ui.xml]: @LarryMatte why do you have `col_t_push_0` and `col_m_push_0` here? It looks like bad code to me, as there is no push for desktop and push_0 is supposed to overwrite a previous push ... (this comment goes for next items having the same pattern)

File: src/main/java/com/arcbees/website/client/application/support/SupportView.ui.xml [hunk 8]
```diff
@@ -215,20 +221,22 @@
...
                         </h3>
                         <p>
-                            <ui:msg description="Community Resources - Paragraph -">Get answers promptly from Stack
+                            <ui:msg description="Community Support - Paragraph -">Get answers promptly from Stack
                                 Overflow, our Google Groups and our Google+ Communities, with or without a package!
                             </ui:msg>
                         </p>
                     </div>
-                    <div class="{res.grid.col} {res.grid.col_1_3} {res.grid.col_push_1} {res.grid.col_t_1_3} {res.grid.col_t_push_0} {res.grid.col_m_full} {res.grid.col_m_push_0} {page.style.responseTime}">
+                    <div class="{res.grid.col} {res.grid.col_1_3} {res.grid.col_push_1} {res.grid.col_t_1_3}
```
Comment (jasonlemay) [inline src/main/java/com/arcbees/website/client/application/support/SupportView.ui.xml]: @LarryMatte Like here, there's a push_0 but it overwrites the `col_push_1`

File: src/main/java/com/arcbees/website/client/application/support/SupportView.ui.xml (hunk 7 above)
Comment (LarryMatte) [inline src/main/java/com/arcbees/website/client/application/support/SupportView.ui.xml]: my bad

File: src/main/java/com/arcbees/website/client/application/support/ScrollToAnimation.java (hunk 6 above)
Comment (meriouma) [inline src/main/java/com/arcbees/website/client/application/support/ScrollToAnimation.java]: Yeah, should be extracted to an `arcbees-ui` repository, with the slider

File: src/main/resources/com/google/gwt/i18n/client/LocalizableResource_fr.properties [hunk 9]
```diff
@@ -718,15 +726,18 @@ C2CAD1457536032C387B80EBEFEAE09D=GAE Studio vous aide à optimiser vos applicati
...
 
 # Description: Chose what you want to see
+# TODO: DEPRECATED (CONSIDER REMOVING)
 CDC8904A3656A43B67DD5C70EEAD6F35=Choisissez ce que vous voulez voir
 
 # Description: Using real-time analysis of datastore requests, GAE Studio...
 DEED6AA7544D8096C42D4C7EBEFD674B=Utilisant l''analyse en temps réel des requêtes des bases de données, GAE Studio cible rapidement celles qui sont inefficaces et fournit des indicateurs de performance aux développeurs afin qu''ils puissent réduire les temps de latence.
 
 # Description: Your metrics at work
+# TODO: DEPRECATED (CONSIDER REMOVING)
```
Comment (meriouma) [inline src/main/resources/com/google/gwt/i18n/client/LocalizableResource_fr.properties]: Was this really removed? If yes then you can remove this translation

File: src/main/resources/com/google/gwt/i18n/client/LocalizableResource_fr.properties [hunk 10]
```diff
@@ -718,15 +726,18 @@ C2CAD1457536032C387B80EBEFEAE09D=GAE Studio vous aide à optimiser vos applicati
...
 E7AE036C57760808BA340B15D3439317=Indicateur de performance
 
 # Description: Struggling with data editing?
+# TODO: DEPRECATED (CONSIDER REMOVING)
```
Comment (meriouma) [inline src/main/resources/com/google/gwt/i18n/client/LocalizableResource_fr.properties]: idem

**Comment by meriouma** [comment]
LGTM

File: src/main/resources/com/google/gwt/i18n/client/LocalizableResource_fr.properties (hunk 9 above)
Comment (LarryMatte) [inline src/main/resources/com/google/gwt/i18n/client/LocalizableResource_fr.properties]: yep, well the gae page in now visually the same as the other products pages, this was the text in the btn to open the "drawer" in the old visual, it should be rmv when I pull master

**Pull request merged**